{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Data Risk Manager",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "2.0.6",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "datePublic": "2020-09-21T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM Data Risk Manager (iDNA) 2.0.6 could allow a privileged user to cause a denial of service due to improper input validation. IBM X-Force ID: 184937."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "BM": {
                            "A": "H",
                            "AC": "L",
                            "AV": "N",
                            "C": "N",
                            "I": "L",
                            "PR": "H",
                            "S": "U",
                            "SCORE": "5.500",
                            "UI": "N"
                        },
                        "TM": {
                            "E": "U",
                            "RC": "C",
                            "RL": "O"
                        }
                    }
                }
            ],
            "problemtypes": [
                {
                    "descriptions": [
                        {
                            "description": "Denial of Service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-12-15",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "name": "https://www.ibm.com/support/pages/node/6335281",
                    "url": "https://www.ibm.com/support/pages/node/6335281"
                },
                {
                    "name": "ibm-idna-cve20204618-dos (184937)",
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/184937"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "psirt@us.ibm.com",
                    "DATE_PUBLIC": "2020-09-21T00:00:00",
                    "ID": "CVE-2020-4618",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "Data Risk Manager",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "2.0.6"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "IBM"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "IBM Data Risk Manager (iDNA) 2.0.6 could allow a privileged user to cause a denial of service due to improper input validation. IBM X-Force ID: 184937."
                        }
                    ]
                },
                "impact": {
                    "cvssv3": {
                        "BM": {
                            "A": "H",
                            "AC": "L",
                            "AV": "N",
                            "C": "N",
                            "I": "L",
                            "PR": "H",
                            "S": "U",
                            "SCORE": "5.500",
                            "UI": "N"
                        },
                        "TM": {
                            "E": "U",
                            "RC": "C",
                            "RL": "O"
                        }
                    }
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "Denial of Service"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://www.ibm.com/support/pages/node/6335281",
                            "refsource": "CONFIRM",
                            "title": "IBM Security Bulletin 6335281 (Data Risk Manager)",
                            "url": "https://www.ibm.com/support/pages/node/6335281"
                        },
                        {
                            "name": "ibm-idna-cve20204618-dos (184937)",
                            "refsource": "XF",
                            "title": "X-Force Vulnerability Report",
                            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/184937"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2020-4618",
        "datePublished": "2020-09-21T00:00:00",
        "dateUpdated": "2021-12-15",
        "state": "PUBLIC"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}